plusplusbot/commands/scorekeeper_commands/leaderboard_command.py

```diff
@@ -17,8 +21,21 @@ def __init__(self, *args, **kwargs):
 
     def execute(self):
         yield from super().execute()
-
-        leaderboard = self.scorekeeper.leaderboard(self.args["channel"])
+        leaderboard = []
+
+        time_unit = self.args.get('range')
```

Comment:
Same in this file with the mixed single and double quotes, if we can move to just double quotes, will make my brain happy